docs/adr/0011-operators.md

```diff
@@ -0,0 +1,145 @@
+# Managing operators
+
+- Status: accepted
+- Deciders: tumido, HumairAK, 4n4nd, anishasthana
+- Date: 2021-01-27
+
+Technical Story: [issue-1](https://github.com/operate-first/apps/issues/126), [pr-1](https://github.com/operate-first/apps/pull/131), [pr-2](https://github.com/operate-first/apps/pull/153)
+
+## Context and Problem Statement
+
+A `cluster-scope` application definition introduced by [ADR 0010][] is a bit vague for use cases like deploying operators. Operator Lifecycle Manager (OLM) defines a resource overlap where a single resource kind can be used in different contexts which may go against the distinction introduced in [ADR 0010][]. This document should provide guidance to how each of the resource kinds for OLM should be treated in each context and usage variant we aim to support.
```

Comment:
> defines a resource overlap where a single resource kind can be used in different contexts 

Not sure what this means, can you explain?

Reply:
For example: `Subscription` resource is deploying a specific operator which can be either cluster-scoped or namespace-scoped. Hence the overlap, because a single namespaced resource can be treated multiple different ways and we need to define in which case it goes to `cluster-scope` and in what other case it goes elsewhere...

If you have a better wording, than "this is a resource overlap", please help me out here.

Reply:
Hrmm yeah, I'd suggest breaking it up and including your explanation in the reply, maybe something like: 

> Operator Lifecycle Manager (OLM) defines a resource overlap where a single resource kind can be used in different contexts. For example, `Subscription` resource can subscribe to  a specific operator which can be either cluster-scoped or namespace-scoped. This may go against the distinction introduced in ADR 0010.

Reply:
that sounds nice. Applied :+1: